api/mailer/SceGoogleApiHandler.js

```diff
@@ -147,6 +147,52 @@ class SceGoogleApiHandler {
       }
     });
   }
+
+  /**
+   * gabs all the events from the calendar id passed into the method,
+   * will log the next 10 events into the console.
+   * @param calendarId  calendar id specified for which calendar to pull from
+   */
+  getEventsFromCalendar(calendarId) {
```

Comment:
instead of console.logging the events array I would just return it, after all we will eventually build an API for this function